index.html

```diff
@@ -285,18 +285,6 @@
           publisher: "ITU",
           date: "2021-04",
           href: "https://www.itu.int/ITU-T/recommendations/rec.aspx?rec=14652"
-        },
-        "MovieLabs-Recommended-Best-Practice-for-SDR-to-HDR-Conversion": {
```

Comment:
I think there is an issue in that the MovieLabs best practice is slightly controversial and isn't used by all manufacturers or broadcasters.  By doubling the brightness, you perceive more detail in shadow regions and make the overall appearance brighter.  There are alternative schemes which maintain the perceptual look and the differences and when to use which scheme are detailed in ITU reports.

Reply:
I don't think that the broadcast specific text aids implementers of PNG encoder or decoder libraries nor does it expand knowledge on who the new chunks are to be used.  The text also contains a lot of terminology which is specific such as shading, single-master* which would need defining and, for single-master, is again has multiple settings in use around the world.
The ITU appendix linked does not exist.
Perhaps we should keep these (MovieLabs and ITU reports) in the bibliography rather than the main text.

Reply:
* ISO is moving away from using the term Master and I have seen suggestions SMPTE and ITU will follow.

Reply:
> we shouldn't make it broadcast-irrelevant either

I think relevance is already well established, through ITU references such as BT.2100 and BT.2020